bin/condor_submit

```diff
@@ -67,6 +67,11 @@ def main():
     - launch
     """
 
+    # set up to use our custom condor_vault_storer until we get
+    # the updated one in the condor release
+    #
+    bindir = os.path.abspath(os.path.dirname(__file__))
+    os.environ["_condor_SEC_CREDENTIAL_STORER"] = f"{bindir}/condor_vault_storer"
```

Comment:
So, yes this *does* override that setting that POMS users have been using, on purpose, because that POMS setting prevents token-weakening from happening. 
The new condor_vault storer works in the POMS case now. 
Once this rolls out, I will be giving amended recommendations for POMS to the POMS developers for what to set in the environment for POMS launches so we  can run with the official htcondor condor_vault_storer.

Reply:
Got it.  Thanks for clarifying that.